packages/style-engine/class-wp-style-engine.php

```diff
@@ -267,6 +267,17 @@ final class WP_Style_Engine {
 					),
 					'path'          => array( 'typography', 'lineHeight' ),
 				),
+				'textAlign'      => array(
+					'property_keys' => array(
+						'default' => 'text-align',
+					),
+					'path'          => array( 'typography', 'textAlign' ),
+					'classnames'    => array(
+						'has-text-align-$slug' => array(
+							'use_value' => true,
```

Comment:
This could also be an enum-type array of valid values, e.g., `array( 'left', 'right', 'center' )` so we can test the validity of incoming style values.

Problem is that the array length could become very large. See possible values for `text-align` https://developer.mozilla.org/en-US/docs/Web/CSS/text-align